src/main/.checkstyle_checks.xml

```diff
@@ -17,74 +17,74 @@
     </module>
     <property name="tabWidth" value="4"/>
     <module name="FileContentsHolder"/>
-    <module name="JavadocStyle">
-      <property name="checkHtml" value="false"/>
-    </module>
-    <module name="LocalFinalVariableName"/>
-    <module name="LocalVariableName"/>
-    <module name="MemberName">
-      <property name="format" value="^(([a-z][a-zA-Z0-9]*$)|(_[A-Z][a-zA-Z0-9]*_[a-z][a-zA-Z0-9]*$))"/>
-    </module>
-    <module name="MethodName">
-      <property name="format" value="^[a-z][a-zA-Z0-9]*$"/>
-    </module>
+    <!--<module name="JavadocStyle">-->
+      <!--<property name="checkHtml" value="false"/>-->
+    <!--</module>-->
```

Comment:
Could you use a single comment <!-- .... --> for a single rule instead of one per line?

Reply:
Or was this commenting done automatically by a CheckStyle plugin?

Reply:
@eregon I did this commenting in the IDE, it's the default comment for `Ctrl + /` in IntelliJ.

Reply:
IntelliJ is not even good at making block comments? :trollface: 
Could you do it via another way or manually?
Otherwise it's gonna be more work to remove the comments and try new rules.

Reply:
Atom does it correctly fwiw.

Reply:
@eregon I think this comment is more effective for example:
```
<!-- rule-1
        rule-2
        rule-3
        rule-4
        rule-5 -->
```
If you want to try rule 3 you'd have to change at least 2 lines with this format.  The other format only changes one line.

It's easy to try new rules, just select the line and do `Ctrl+/` 😉

Reply:
Well except in editors which handle *correctly* block comments and don't split them into line-ish comments.
But I meant one comment per rule, `module` and `property` are the same rule here.
```xml
<module>
  <property/>
</module>

becomes

<!-- <module>
  <property/>
</module> -->

and if I take the current version:

<!--<module>-->
  <!--<property/>-->
<!--</module>-->

it becomes (in Atom):

<module>-->
  <!--<property/>-->
<!--</module>
```

Anyway, it probably doesn't matter much, we can't have both editors agree and we can search/replace as needed.